modules/openthread/platform/platform.h

```diff
@@ -0,0 +1,272 @@
+/*
```

Comment:
As I understand this is OT file why not include it simply? Have You made any changes to it?